test/functionalTest/cases/1187_empty_attribute_in_create/empty_attribute_in_create.test

```diff
@@ -120,11 +120,6 @@ Date: REGEX(.*)
         "type": null,
         "value": ""
     },
-    "A2": {
```

Comment:
You are right... Actually, the fix is correct.

Maybe it would be clerarer chaning a little bit the step descriptions. In particular:

03. POST /v2/entities to create E1/A2="", using json object

to

03. POST /v2/entities to create E1/A2="", using json object (fails, as entity already exists)

And:

04. GET /v2/entities/E1 to see A2==""

to

04. GET /v2/entities/E1 to not see A2